lib/ueberauth/strategy/okta.ex

```diff
@@ -145,8 +146,10 @@ defmodule Ueberauth.Strategy.Okta do
   """
   def handle_callback!(%Conn{params: %{"code" => code}} = conn) do
     module = option(conn, :oauth2_module)
+    opts = Keyword.merge(conn.private.ueberauth_request_options.options, redirect_uri: callback_url(conn))
```

Comment:
Is it worth just passing the whole conn? I'm good either way

Reply:
If you do not mind, I would rather keep it like so; i.e passing just what we need.